examples/seesaw_rotary_neopixel.py

```diff
@@ -0,0 +1,47 @@
+"""I2C rotary encoder NeoPixel color picker and brightness setting example."""
+import board
+from adafruit_seesaw import seesaw, neopixel, rotaryio, digitalio
+try:
+    import _pixelbuf
+except ImportError:
+    import adafruit_pypixelbuf as _pixelbuf
+
+# For use with the STEMMA connector on QT Py RP2040
+# import busio
```

Comment:
Is busio the thing to import here? This is actually a point I'm trying to get a better handle on--when and when not to import busio

Reply:
`busio` only need to be imported if you have to specify individual I2C pins or I2C pins _other_ than SCL and SDA. Otherwise, use `board.I2C()` which is basically an object representing `board.SCL, board.SDA`. In the case of the QT Py RP2040 STEMMA connector, it's SDA1 and SCL1, so it would need `busio`.

Reply:
Ah, that clears it up, I was afraid I needed to worry about clock timing issues on boards and I just didn't know how to begin knowing when and when not to use it. 

Thank you for the explanation!